app/controllers/price_policies_controller.rb

```diff
@@ -168,7 +168,7 @@ def build_duration_rates
         duration_rate = price_policy.duration_rates.select { |dr| dr.rate_start_index == rs_index.to_s }.first
 
         if duration_rate.nil?
-          dr = price_policy.duration_rates.build rate_start_index: rs_index
+          dr = price_policy.duration_rates.build rate_start_index: rs_index, rate_start_id: rate_start.id
```

Comment:
@LeticiaErrandonea with this change I can save price rules and rate starts, as long as there are no values entered for duration rates.  So.... progress?